### PR TITLE
Create ui-c8y-1021-0-0-upgrade-to-angular-18-6833.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by DawidSAG

Ticket: [MTM-60178](https://cumulocity.atlassian.net/browse/MTM-60178)
Original PR: [6833](https://github.com/Cumulocity-IoT/cumulocity-ui/pull/6833)

[MTM-60178]: https://cumulocity.atlassian.net/browse/MTM-60178?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1021-0-0-upgrade-to-angular-18-6833.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Upgrade to Angular 18 (#6833)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1021-0-0-upgrade-to-angular-18-6833.md]: The Claude AI suggested to use the following title:
```suggestion
title: Upgrade to Angular 18
```

File: content/change-logs/application-enablement/ui-c8y-1021-0-0-upgrade-to-angular-18-6833.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-60178
+version: 1021.0.0
+---
+Upgrade to Angular 18 (#6833)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1021-0-0-upgrade-to-angular-18-6833.md]: The Claude AI suggested to use the following body:
```suggestion
As part of keeping the Cumulocity IoT platform up-to-date with the latest web technologies, Cumulocity IoT has been upgraded to use Angular 18. This upgrade brings the latest features, performance improvements, and bug fixes from the Angular framework to the Cumulocity IoT platform. The upgrade to Angular 18 should be largely transparent to users, with no visible changes to the user interface or existing functionality. However, any custom widgets or plugins developed for previous versions of Cumulocity IoT may need to be updated to ensure compatibility with Angular 18.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Comment by reey** [comment]
Do we need both?
https://github.com/Cumulocity-IoT/c8y-docs/pull/2534

File: content/change-logs/application-enablement/ui-c8y-1021-0-0-upgrade-to-angular-18-6833.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-60178
+version: 1021.0.0
+---
+As part of keeping the Cumulocity IoT platform up-to-date with the latest web technologies, Cumulocity IoT has been upgraded to use Angular 18. This upgrade brings the latest features, performance improvements, and bug fixes from the Angular framework to the Cumulocity IoT platform. The upgrade to Angular 18 should be largely transparent to users, with no visible changes to the user interface or existing functionality. However, any custom widgets or plugins developed for previous versions of Cumulocity IoT may need to be updated to ensure compatibility with Angular 18.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1021-0-0-upgrade-to-angular-18-6833.md]:
```suggestion
As part of keeping the {{< product-c8y-iot >}} platform up-to-date with the latest web technologies, {{< product-c8y-iot >}} has been upgraded to use Angular 18. This upgrade brings the latest features, performance improvements, and bug fixes from the Angular framework to the {{< product-c8y-iot >}} platform. The upgrade to Angular 18 should be largely transparent to users, with no visible changes to the user interface or existing functionality. However, any custom widgets or plugins developed for previous versions of {{< product-c8y-iot >}} may need to be updated to ensure compatibility with Angular 18.
```

**Review by BeateRixen — APPROVED** [review]

**Comment by BeateRixen** [comment]
@janhommes or @rahultsag please approve

**Review by rahulc8y — APPROVED** [review]

**Pull request merged**